xml/obs_build_containers.xml

```diff
@@ -91,4 +93,170 @@
+           instance is reachable
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$PROJECT</literal>: The name of the project where the image is
+           build, with all colons replaced with forward slashes.
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$REPOSITORY</literal>: The name of the repository where the
+           containers are published.
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$IMAGE_NAME</literal>: The name of this container image. It
+           defaults to the name of the package from which the container is
+           build. Alternatively, a different image name can be specified in the
+           build recipe, e.g. via the <literal>containerconfig</literal> element in a
+           &kiwi; file (see <link
+           xlink:href="https://osinside.github.io/kiwi/building_images/build_docker_container.html?highlight=containerconfig">the
+           kiwi documentation</link> for further details).
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$TAG</literal>: The image tag. This defaults to
+           <literal>latest</literal> with alternatives being provided in a similar
+           fashion as the image's name.
+         </para>
+       </listitem>
+     </itemizedlist>
+   </para>
+   <para>
+     The <link
+     xlink:href="https://github.com/openSUSE/cooverview">cooverview</link>
+     project provides a simple user facing webpage to search for containers
+     published by the Open Build Service and can be used to conveniently obtain
+     the correct registry URLs. It is used to power <link
+     xlink:href="https://registry.opensuse.org/">registry.opensuse.org</link>
+   </para>
+ </sect2>
+
+ <sect3>
+   <title>
+     Container Image Signatures
+   </title>
+   <para>
+     The Open Build Service automatically signs every package that has been
+     build and publishes the cryptographic signature alongside with
+     it. Container images are no exception to this and the detached signatures
+     can be used by <link xlink:href="https://podman.io/">podman</link> to
+     verify every image that is pulled from the registry.
+   </para>
+   <para>
+     Podman has to be configured first as outlined in the following steps.
+     <itemizedlist>
+       <listitem>
+         <para>
+           Create a <literal>yaml</literal> file under
+           <literal>/etc/containers/registries.d/</literal> with an appropriate
+           name for your instance and the following contents:
+           <example>
+             <title>registry.yaml</title>
+             <screen>
+---
+docker:
+  registry.opensuse.org:
+    sigstore: https://registry.opensuse.org/sigstore
+             </screen>
+           </example>
+           Where you have to replace <literal>registry.opensuse.org</literal>
```

Comment:
1. Shorten:

```suggestion
           Replace <literal>registry.opensuse.org</literal>
```

2. DocBook knows the element `<replaceable>` (shown in italics in our documents) which can be used like this:

```
  <replaceable>REGISTRY_URL</replaceable>:
    sigstore: https://<replaceable>REGISTRY_URL</replaceable>/sigstore

[...]

    Replace <replaceable>REGISTRY_URL</replaceable> with the appropriate URL to your
    instance of &obs; (for example, <literal>registry.opensuse.org</literal>.
```

Reply:
Thanks, this is pretty nifty and I've now included it in another place as well!

I have a tiny gripe with the styling though, the italics make the `/` after `REGISTRY_URL` harder to see imho:
![image](https://user-images.githubusercontent.com/45594031/119147707-1f479d80-ba4c-11eb-9942-ed0855bab81e.png)

Reply:
Right, italics don't always play well with punctuation and in this case make it a bit harder to discern whether the following character is a slash `/` or a pipe `|`. Not sure I have a better idea for formatting right now -- do you?

Reply:
Not really, no. I'd keep it this way, to be consistent with the rest of the documentation.